src/Microsoft.CodeAnalysis.FxCopAnalyzers/Microsoft.CodeAnalysis.FxCopAnalyzers.sarif

```diff
@@ -2358,6 +2358,25 @@
               "Telemetry"
             ]
           }
+        },
+		  "CA2014": {
```

Comment:
The guidance I got in my PR was to not edit the .sarif file.

Reply:
Ok, don't need to update md files too?